lib/src/components/NxTable/__tests__/NxTable.test.tsx

```diff
@@ -0,0 +1,17 @@
+/*
+ * Copyright (c) 2019-present Sonatype, Inc.
+ * This program and the accompanying materials are made available under
+ * the terms of the Eclipse Public License 2.0 which accompanies this
+ * distribution and is available at https://www.eclipse.org/legal/epl-2.0/.
+ */
+import React from 'react';
+import { shallow } from 'enzyme';
+import NxTable from '../NxTable';
+
+describe('NxTable', function() {
+  it('renders a table with the expected class names', function() {
+    const component = shallow(<NxTable className="test" />);
+
+    expect(component).toMatchSelector('table.nx-table.test');
+  });
```

Comment:
Test pass-through of some other props

Reply:
done

Reply:
> done

I'm not seeing it

Reply:
Passing children is now covered by the 'computes the columns in NxTableHead and passes them to NxTableBody' test. Though I suppose I didn't add one to check that the id is passed through.

Reply:
88c798c840845469dec8d575fff01f160d84db4a